components/core/Application.js

```diff
@@ -209,6 +210,7 @@ export default class ApplicationPage extends React.Component {
         resource: this.props.resources.pubsub,
         viewer: this.state.viewer,
         onUpdate: this._handleUpdateViewer,
+        onNewActiveUser: this._handleActiveUsers,
```

Comment:
`onNewActiveUser: this._handleNewActiveUser` would be the convention I like. You can change this later.

Reply:
Sure. I'll change it now.